package.js

```diff
@@ -3,13 +3,20 @@ Package.describe({
   name: "practicalmeteor:mocha",
   summary: "Write package tests with mocha and run them in the browser or from the command line with spacejam.",
   git: "https://github.com/practicalmeteor/meteor-mocha.git",
-  version: '2.1.0_7'
+  version: '2.1.1-rc.1',
+  testOnly: true
+});
+
+
+Npm.depends({
```

Comment:
I know, but we made some modifications to mocha, to adapt it to meteor, and we also found in both chai, sinon and loglevel that using the browserified one in both client and server reduces build times, so it's a practice we've been using, instead of doing Npm.depends server side and adding the browserified one client side. It also allows us to modify things only in one place, if needed, in the browserified one. But we probably should reevaluate this with the new modules support in 1.3